code/game/machinery/autolathe.dm

```diff
@@ -228,7 +228,8 @@
 						materials.use_amount(materials_used)
 						for(var/i=1, i<=multiplier, i++)
 							var/obj/item/new_item = new being_built.build_path(T)
-							new_item.materials = materials_used.Copy()
+							new_item.materials[MAT_METAL] = materials_used[MAT_METAL] / multiplier
+							new_item.materials[MAT_GLASS] = materials_used[MAT_GLASS] / multiplier

```

Comment:
See if there's a function in the materials datum to handle this better though, this copypaste isn't supposed to happen.